<launch>
    <!-- map -->
    <!-- <node name="map_odom" pkg="tf2_ros" type="static_transform_publisher" args="0 0 0 0 0 0 1 map odom"/> -->
    <arg name="map_file" default="$(find astar_nav)/map/map_empty2/new_map.yaml"/>

    <!-- Run the map server -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

    <!-- rviz -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find stage_simulation)/rviz/stage.rviz" />
</launch>
